README.md

```diff
@@ -3,6 +3,12 @@ Pi Video Machine - a scalable, synchronized, and networked-controlled, raspberry
 
 
 ### { This is a work in progress }
+  
+- TODO: rename the variable
+- TODO: rewrite logging
+- TODO: rewrite logic between commands
+- TODO: Create another python file to control two display
```

Comment:
this sounds more like a latter version of the project - the end goal (last milestone) -, rather than an alternative version? 
things to decide when we get to that point include:
- is it using 2 UDP servers on different ports?
- is it possible to detect with rpi device has 1 or 2 screens connected?

of course, I am not saying that you shouldn't start working on it, I am just saying that it might turn out to be *the* final version of the project, and not an alternative python script 😄